v0/destinations/am/utils.js

```diff
@@ -61,10 +61,20 @@ function getPlatform(payload, sourceKey) {
     : payloadVal;
 }
 
+function getBrand(payload, sourceKey) {
+  const payloadVal = get(payload, sourceKey);
+
+  if (payload.channel && payload.channel.toLowerCase() === "web") {
+    return getInfoFromUA("device.brand", payload, payloadVal);
```

Comment:
Do we know for sure, this information will be part of UA?

Reply:
@abandonned AFAIK `ua` will not contain the brand information so we can skip this part. Even for web source payload, we have to have the device brand info in `context.device`

Lets make it clear in our docs what AM takes for computing `device_family` and from where we are picking those components from our payload

Reply:
Updated as per latest discussion. @utsabc @saikumarrs